Library/Homebrew/language/python.rb

```diff
@@ -70,7 +70,7 @@ def self.in_sys_path?(python, path)
       quiet_system python, "-c", script
     end
 
-    def self.setup_install_args(prefix)
+    def self.setup_install_args(prefix, python = "python3")
```

Comment:
```suggestion
    def self.setup_install_args(prefix, python: "python3")
```

maybe? No strong views either way.

Reply:
I agree that this is better, but it's not really consistent with the rest of the API in `language/python.rb`. Everything else seems to use `python =` rather than `python: `.